Makefile

```diff
@@ -36,6 +36,7 @@ test:
 	CACHE_ARGS=""; \
 	if [[ -n "$(DOCKER_READ_CACHE_ORG)" ]]; then \
 		CACHE_ARGS="$${CACHE_ARGS} --cache-from type=registry,ref=$(DOCKER_READ_CACHE_ORG)/plugins-$*-base-build:buildcache"; \
+		$(DOCKER) pull $(DOCKER_READ_CACHE_ORG)/plugins-$*-base-build:latest || :; \
```

Comment:
Should we pick one or the other, i.e., either docker pull the latest image _or_ `--cache-from` ?